### ISSUE TITLE
Giotto cell-cell interaction analysis

### ISSUE THREAD

**Issue by evenDDDDD** [giotto-suite/Giotto#140]
Hey,
Firstly，thanks for the great tool !

Can Giotto support cell-cell interaction analysis or ligand-receptor interaction analysis on merfish data？

Sincerely looking forward to your reply！

**Comment by RubD** [comment]
Yes, that should all be possible. Let me know if you have any specific questions or difficulties.
For the ligand-receptor analysis it would depend on which ligand-receptor combinations are available in your merfish dataset.
